### ISSUE TITLE
`Flutter doctor` not generated in Ubuntu

### ISSUE THREAD

**Issue by BrianCraig** [werainkhatri/issue#2]
### Describe the bug

Ubuntu didn't generate `flutter doctor -v`
### Steps to reproduce

1. run issue creation on Ubuntu.

### Expected behavior

2. See flutter information on the issue.

### Actual results

2. No flutter information.

### Additional context

[Screencast from 18-10-22 16:11:18.webm](https://user-images.githubusercontent.com/4956754/196523187-380f68c7-a7c6-40f3-ba79-a1b24eb585ca.webm)

###### This issue was created using [issue](https://pub.dev/packages/issue).

<details>
<summary>Flutter doctor -v</summary>

```
[✓] Flutter (Channel stable, 3.3.4, on Ubuntu 22.04.1
    LTS 5.15.0-50-generic, locale en_US.UTF-8)
    • Flutter version 3.3.4 on channel stable at
      /home/bri/sdk/flutter-stable
    • Upstream repository
      https://github.com/flutter/flutter.git
    • Framework revision eb6d86ee27 (2 weeks ago),
      2022-10-04 22:31:45 -0700
    • Engine revision c08d7d5efc
    • Dart version 2.18.2
    • DevTools version 2.15.0

[✓] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
    • Android SDK at /home/bri/Android/Sdk
    • Platform android-32, build-tools 32.1.0-rc1
    • Java binary at: /home/bri/app/android-studio/jre/bin/java
    • Java version Java(TM) SE Runtime Environment (build 17.0.1+12-LTS-39)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Linux toolchain - develop for Linux desktop
    • Ubuntu clang version 14.0.0-1ubuntu1
    • cmake version 3.22.1
    • ninja version 1.10.1
    • pkg-config version 0.29.2

[✓] Android Studio (version 2021.2)
    • Android Studio at /home/bri/app/android-studio
    • Flutter plugin version 70.2.2
    • Dart plugin version 212.5744
    • Java version Java(TM) SE Runtime Environment (build 17.0.1+12-LTS-39)

[✓] VS Code (version 1.71.2)
    • VS Code at /usr/share/code
    • Flutter extension version 3.50.0

[✓] Connected device (2 available)
    • Linux (desktop) • linux  • linux-x64      • Ubuntu 22.04.1 LTS 5.15.0-50-generic
    • Chrome (web)    • chrome • web-javascript • Google Chrome 105.0.5195.125

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
</details>

**Comment by werainkhatri** [comment]
the `issue.dart` class in example uses a `SimpleBugReportIssueTemplate`, 

https://github.com/werainkhatri/issue/blob/5bbfb4e3d1ac0e0c6f76aba99a205fc95543b3ec/example/issue.dart#L5

which does not include `FlutterDoctorIssueSection`,

https://github.com/werainkhatri/issue/blob/5bbfb4e3d1ac0e0c6f76aba99a205fc95543b3ec/lib/src/models/issue_template.dart#L70-L94

unlike `FlutterBugReportIssueTemplate`, which does

https://github.com/werainkhatri/issue/blob/5bbfb4e3d1ac0e0c6f76aba99a205fc95543b3ec/lib/src/models/issue_template.dart#L104-L131


I realize the discrepancy, which was intentional while coding the package, but the intention was precedent on an incorrect assumption. fixing asap!

thanks for pointing that out.

**Comment by werainkhatri** [comment]
fixed in version `0.0.4`. closing...

feel free to reopen if you feel otherwise.
